config/install/core.entity_form_display.node.landing_page.default.yml

```diff
@@ -22,6 +22,7 @@ dependencies:
     - field.field.node.landing_page.field_landing_page_show_contact
     - field.field.node.landing_page.field_landing_page_summary
     - field.field.node.landing_page.field_metatags
+    - field.field.node.landing_page.field_node_display_headings
```

Comment:
Any reason for the `node` part of the name `field_node_display_headings`?

Reply:
@anthony-malkoun yes, So I remember a ticket from Kurt that making the field name more generalized which is used in different content types. So this fields storage sits in tide_core and the field instances in the tide_ modules, so I gave `node` instead of `field_landing_page`